pkg/rbac/rbac.go

```diff
@@ -80,14 +80,14 @@ type User struct {
 // This informer reloads the policy whenever the ConfigMap is updated.
 func refreshEnforcerInBackground(
 	ctx context.Context,
-	kubeClient kubernetes.KubernetesConnector,
+	kubeConnector kubernetes.KubernetesConnector,
 	enforcer *casbin.Enforcer,
 	l *zap.SugaredLogger,
 ) error {
 	inf, err := informer.New(
-		informer.WithConfig(kubeClient.Config()),
+		informer.WithConfig(kubeConnector.Config()),
```

Comment:
Informer uses controller-runtime cache under the hood. The PR description states `Caching functionality is completely disabled so far`. Does it affect the informer?